src/libraries/Microsoft.Bcl.Memory/src/Microsoft.Bcl.Memory.csproj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum);$(NetCoreAppCurrent)</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <UseCompilerGeneratedDocXmlFile>false</UseCompilerGeneratedDocXmlFile>
```

Comment:
Why disable the compiler generated doc xml file?

Reply:
Yep - keep it enabled.  Best to use the source-authored docs here.

Reply:
Sure, just copied from existing Microsoft.Bcl.* packages